schemas/com.snowplowanalytics.snowplow/media_player_event_seek_end/jsonschema/1-0-0

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event sent when a seek operation completes.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_event_seek_end",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "label": {
```

Comment:
Seek position, or is this captured globally?

Reply:
We capture the position as the `currentTime` property in the `media_player` schema.